pages/linux/fdisk.md

```diff
@@ -0,0 +1,7 @@
+# fdisk
+
+> Manipulate disk partition table.
+
+- View all available partitions:
```

Comment:
Add the option to change partition table - `fdisk {{/dev/sda}}` first.

Reply:
Ok fine, I added that but as a second option. Example of editing before listing does not make any sense.

Reply:
Agree.